README.rst

```diff
@@ -166,51 +166,57 @@ Usage
 .. code:: console
 
     $ git-big-picture --help
-    Usage: git-big-picture OPTIONS [<repo-directory>]
-
-    Options:
-    --version             show program's version number and exit
-    -h, --help            show this help message and exit
-    --pstats=FILE         run cProfile profiler writing pstats output to FILE
-    -d, --debug           activate debug output
-
-    Output Options:
-        Options to control output and format
-
-        -f FMT, --format=FMT
-                            set output format [svg, png, ps, pdf, ...]
-        -g, --graphviz      output lines suitable as input for dot/graphviz
-        -G, --no-graphviz   disable dot/graphviz output
-        -p, --processed     output the dot processed, binary data
-        -P, --no-processed  disable binary output
-        -v CMD, --viewer=CMD
-                            write image to tempfile and start specified viewer
-        -V, --no-viewer     disable starting viewer
-        -o FILE, --outfile=FILE
+    usage: git-big-picture OPTIONS [REPOSITORY]
+
+    positional arguments:
+      REPOSITORY            path to the Git working directory
```

Comment:
What if the repo is bare? IIRC it should work too?

Reply:
I think so:

```console
# cd "$(mktemp -d)"
# git init
# git commit --allow-empty -m init
[master (root-commit) 0c98926] init

# git --no-pager log --oneline
0c98926 (HEAD -> master) init

# cd .git/
# git --no-pager log --oneline
0c98926 (HEAD -> master) init
```

Reply:
I'm not sure though if we should "advertise" the bare repo variation, though. It kind of works by accident.

Reply:
sounds good,  it's an edge use-case anyway.